index.html

```diff
@@ -2,16 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me FCC: Test Suite Template</title>
```

Comment:
- It's good practice to add the viewpoint for responsive web design across different devices in the head tag. 😃

Reply:
This is a nice comment however, It would be better to provide additional information about this. Providing a link to where the author could read more about this would help the author to make research and be able to implement the required changes.